### PR TITLE
Added a test for comment view

### PR DESCRIPTION
**Pull request by pt-arvind**
https://github.com/code-corps/code-corps-api/issues/242

Checking vs CI to see if the tests run OK. Should be 336 total tests with this one.

### CODE REVIEW

File: test/views/comment_view_test.exs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          "updated-at" => comment.updated_at
+        },
+        relationships: %{
+          "user" => %{
+            data: %{id: comment.user_id |> Integer.to_string,
+                    type: "user"}
+          },
+          "post" => %{
+            data: %{id: comment.post_id |> Integer.to_string,
+                    type: "post"}
```
Comment (begedin) [inline test/views/comment_view_test.exs]: Nitpicky, I know, but we actually prefer the following format:

```
data: %{
  id: comment.post_id |> Integer.to_string,
  type: "post"
}
```

**Review by begedin — APPROVED** [review]
Almost perfect. Just a minor thing you could change and then this is good to go.

**Review by begedin — CHANGES_REQUESTED** [review]
Previous review was accidentally approved. Will re-approve after changes.

**Comment by joshsmith** [comment]
@pt-arvind also still want to address why your tests are failing. You could try to find someone in `#code-corps-api` channel on Slack who can pair. I'm a little slammed next two days, unfortunately.

**Comment by joshsmith** [comment]
@pt-arvind would you mind looking at this when you get a chance?

**Comment by pt-arvind** [comment]
yes! I will fix it up tonight

**Comment by joshsmith** [comment]
@pt-arvind thank you!

File: test/views/comment_view_test.exs (hunk 1 above)
Comment (pt-arvind) [reply test/views/comment_view_test.exs]: no worries! I noticed the same style in a couple other tests, would you like me to change those to be what you've described as well?

**Comment by pt-arvind** [comment]
am aware of the messed up git history here, I will figure it out before I ask for the final review tonight.

**Comment by joshsmith** [comment]
@pt-arvind I think it has to do with the way you're handing rebase. That's my best guess. Don't use `merge`. We can pair on this if you'd like.

**Comment by pt-arvind** [comment]
nuked this branch because I rebased improperly. Real PR is in the link below:

https://github.com/code-corps/code-corps-api/pull/272

**Pull request closed without merging**